scripts/run_ansible_test

```diff
@@ -34,20 +34,23 @@ collection_path='/usr/share/ansible/collections/ansible_collections'
 case ${USE_VENV} in
     y|yes|true):
         ansible_test="${HOME}/test-python/bin/ansible-test"
-        collection_path="${HOME}.ansible/collections/ansible_collections"
+        collection_path="${HOME}/.ansible/collections/ansible_collections"
         ;;
 esac
 
-mkdir -p ${HOME}/code/ansible_collections/cifmw/general
-rsync -a --delete ${PROJECT_DIR}ci_framework/* ${HOME}/code/ansible_collections/cifmw/general/
-ln -fs ${collection_path}/* ${HOME}/code/ansible_collections/
+mkdir -p ${HOME}/code
+ansible-galaxy collection install git+https://github.com/openstack-k8s-operators/ci-framework.git,cifmw_create_collection -p ${HOME}/code
```

Comment:
this would prevent using the code we're trying to test I think. It would be better to not consume git+https and install from the existing clone

Reply:
yes, I made that change but looks like I forgot to push it, should be there in the next update